test/index.test.js

```diff
@@ -1,6 +1,58 @@
 var test = require('tape')
+var Validate = require('is-my-json-valid')
+var pollSchema = require('../schema/poll')
+var positionSchema = require('../schema/position')
 
-var {isPoll, versionStrings} = require('../')
+var isNormalisedPoll = Validate(pollSchema)
+var isNormalisedPosition = Validate(positionSchema)
+var { CHOOSE_ONE } = require('../v1/types')
+
+var {isPoll, parsePoll, parsePosition, isPosition, versionStrings} = require('../')
+
+test('parsing a v1 position returns an object that is a valid normalised position object', function (t) {
+  var validPosition = {
+    type: 'position',
+    version: 'v1',
+    root: '%t+PhrNxxXkw/jMo6mnwUWfFjJapoPWxzsQoe0Np+nYw=.sha256',
+    details: {
+      type: CHOOSE_ONE,
+      choice: 0
+    }
+  }
+  t.ok(isPosition(validPosition))
+
+  var parsedPosition = parsePosition(validPosition)
+  t.ok(isNormalisedPosition(parsedPosition))
+  t.end()
+})
```

Comment:
the interesting thing I want to test is : where is the case where something is v1 format, but doesn't produce a normalized poll.... 

How do we test that ... or do we always assert that it has been normalised after we parse?